README.md

```diff
@@ -2,13 +2,46 @@
 
 ## Introduction
 
-Goordinator listens for GitHub webhook events, runs their JSON payloads through
-a JQ filter query and triggers actions if the filter matches.
+Goordinator is an event-processor for GitHub events.
+It provides 2 functionalities.
+
+### Configurable Event-Trigger Loop
+
+Goordinator can listen for GitHub webhook events, runs their JSON payloads
+through a JQ filter query and triggers actions if the filter matches.
+The supported actions are:
+- posting a http-request
+- updating a GitHub branch with it's base branch.
 
-The only supported action is currently posting HTTP-Requests.
 All actions are executed in parallel and retried if they fail until a retry
 timeout expired (default: 2h).
 
+### Serialized GitHub Branch Autoupdater
+
+Autoupdate keeps Pull-Requests (PR) updated with their base-branch.
+Pull-Requests are added to a per base-branch-queue and the first pull-request in
+the queue is kept uptodate with it's base-branch.
+If merging the base-branch into the PR branch fails, it's check status becomes
+negative or the PR became stale, updates for it are suspended.
```

Comment:
This sentence is not 100% clear, you seem to be talking about 2 different things, but they're getting mixed up together by the end of the sentence.
My understanding:
* if merging the base branch in the PR fails, a status check will be in failure state and updates will be suspended
* if the PR became stale (maybe clarify what _stale_ actually means), updates will also be suspended
* updates will be resumed once the PR status checks are successful

Perhaps trying to synthesize the workflow via bullet points would help the reader.

Reply:
thx, I changed it and added bullet points as suggested, I hope it's now more clear